### DISCUSSION TITLE
Toast - Center Alignment

### DISCUSSION THREAD

**Discussion by HarryLobster** [xamarin/XamarinCommunityToolkit#1516] [Q&A] [answered]
I find currently no property to set the alignment of the shown message. 
I would like to center it...

**Comment by pictos** [comment]
@VladislavAntonyuk can you give your input here?

**Comment by VladislavAntonyuk** [comment, marked as answer]
You can customise the android Snackbar layout.

We already have a issue for that: https://github.com/xamarin/XamarinCommunityToolkit/issues/1425
